src/components/Dashboard/StaffNotation.vue

```diff
@@ -76,7 +76,7 @@
             <v-list-item
               data-type="registrars-order"
               @click="showRegistrarsOrderDialog()"
-              :disabled="disabled || isAdminFreeze"
+              :disabled="disabled || isAdminFrozen"
```

Comment:
Is this change related to the ticket?

Reply:
No.

`adminFreeze` is the name of the variable, whereas the getter returns the state.

Reply:
Yes, you bet, just clarifying to make sure I wasn't missing something important.